docs/API.md

```diff
@@ -1713,23 +1733,15 @@ type ResourcesRevision struct {
 }
 ```
 
-#### GET *id*/resources/name.stream[-revision]/arch/filename
+#### GET *id*/resources/name[/revision]
 
-Getting from the `/resources` path retrieves a charm resource from the charm
-with the given id. If version is not specified, it retrieves the latest version
-of the resource. The SHA-256 hash of the data is specified in the HTTP response
-headers.
+Getting from the `/resources` path retrieves a charm resource from the charm with the given id. If version is not specified, it retrieves the latest version of the resource. The SHA-256 hash of the data is specified in the HTTP response headers.
 
-#### PUT *id*/resources/[~user/]series/name.stream-revision/arch?sha256=hash
+#### PUT *id*/resources/[~user/]name/revision/?sha256=hash
```

Comment:
Oops, yes. Also dropped the revision portion since the server will decide what revision the resource is at.